tests/db_seed_data.py

```diff
@@ -0,0 +1,79 @@
+from random import choice
+from random import sample
+from string import ascii_uppercase
+from uuid import uuid4
+
+from tests.application_store_json import application_store_json_template
+
+
+def create_rows(number_of_apps_per_round, number_of_funds, number_of_rounds):
+
+    funds = [str(uuid4()) for _ in range(number_of_funds)]
+
+    rounds = [
+        (str(uuid4()) for _ in range(number_of_rounds))
+        for _ in range(number_of_funds)
+    ]
+
+    app_ids = [
+        (str(uuid4()) for _ in range(number_of_apps_per_round))
+        for _ in range(number_of_funds)
+    ]
+
+    app_fund_rounds = zip(app_ids, funds, rounds)
+
+    verbs = ["Save", "Restore", "Refurbish", "Rebuild", "Remodel"]
+
+    adjects = ["old", "humble", "derelict", "vintage", "loved", "beautiful"]
+
+    places = [
+        "skatepark",
+        "pub",
+        "cinema",
+        "community centre",
+    ]
+
+    cities = [
+        "Bath",
```

Comment:
Can we use fixed data rather than random generation? (So we can rely on these records for testing known values.)